src/swarm/neo/client/mixins/RequestCore.d

```diff
@@ -222,6 +272,31 @@ public template RequestCore ( RequestType request_type_, ubyte request_code,
         ***********************************************************************/
 
         public RequestId request_id;
+
+        /***********************************************************************
+
+            Helper function to properly serialize user-specified params for the
+            request into the `user_params` field. An intermediary serialzation
+            buffer is used to safely convert from the const params provided by
+            the user to the mutable params required internally (in order to be
+            able to deserialize into a Context instance).
+
+            Params:
+                params = user-specified request parameters to store
+                serialize_buffer = intermediary serialization buffer
+
+        ***********************************************************************/
+
+        public void setUserSpecifiedParams ( Const!(UserSpecifiedParams) params,
+            ref ubyte[] serialize_buffer )
+        {
+            enableStomping(serialize_buffer);
```

Comment:
I think you can save on `enableStomping` call here, as the only time you change the length of it is below.